CONTRIBUTING.md

```diff
@@ -28,6 +28,16 @@ If you have questions about one of the issues, please comment on them, and one o
 
 Please provide as much context as possible when you open an issue. The information you provide must be comprehensive enough to reproduce that issue for the assignee. Therefore, contributors may use but aren't restricted to the issue template provided by the project maintainers.
 
+### Guidance for closing issues
+* As per the [ProjectCharter](https://github.com/camaraproject/Governance/blob/5fc802713d71a51da64136f692b16ed620eeffa5/ProjectCharter.md), The default decision making mechanism for the CAMARA Project is [lazy consensus](https://couchdb.apache.org/bylaws.html#lazy). This means that any decision on technical issues is considered supported by the team as long as nobody objects based on substantiated technical grounds.
+* Once consensus is achieved, an issue may be closed by the original author or by a maintainer in the sub-project.
+* Closing an issue should include a comment on how it was resolved, or reference to consensus that the issue can not be resolved.
+* Issues that are considered stale/inactive may be closed, but must include a reference to minutes where it has been agreed to be a stale issue. 
+* Maintainers may not close their own issues without reference to consensus achieved.
```

Comment:
Why not? If even a non-Maintainer is allowed to do that? (see line 33). Propose to omit the line.

Reply:
Yes, will omit the line, as line 33 covers it.